google/cloud/security/iam/__init__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
The [pattern](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/google/cloud/security/scanner/scanner.py) to date has been no blank-line between the license and the doc. Please fix throughout.